bddl/activity_definitions/polishing_shoes/problem0.bddl

```diff
@@ -3,19 +3,19 @@
 
     (:objects
         stain.n.01_1 - stain.n.01
-    	shoe.n.01_1 shoe.n.01_2 - shoe.n.01
```

Comment:
For some of these tasks, maybe it's okay to leave it be `shoe.n.01`? What do you think?